app/frontend/src/assets/languages/locale-fi.json

```diff
@@ -96,12 +96,13 @@
   "sitnet_exam_room_address_street": "Katuosoite",
   "sitnet_exam_room_address_zip": "Postinumero",
   "sitnet_exam_room_machines": "Tenttitilan koneet",
-  "sitnet_exam_rooms": "Tenttitilat",
```

Comment:
Nää muutokset pitää muistaa jotenkin päivittää sinne kielipakettiin (google drivessa). Se ei riitä että muutetaan näitä json-fileitä.